tests/pageObjects/LoginPage.js

```diff
@@ -0,0 +1,23 @@
+class LoginPage{
+    constructor(){
+        this.loginBtnLocator = "input[value='Login']";
+        this.emailFieldLocator = "#spree_user_email";
+        this.passwordFieldLocator = "#spree_user_password";
+        this.alertLocator = "div.alert-danger"
+    }
+
+    async navigate(){
+        await page.goto("http://localhost:3000/admin/login");
+    }
+
+    async enterCredentials(email,password){
+        await page.locator(this.emailFieldLocator).fill(email);
+        await page.fill(this.passwordFieldLocator, password);   
+    }
+
+    async clickLoginBtn(){
+        await page.locator(this.loginBtnLocator).click();
+    }
```

Comment:
You can combine these functions into one
```suggestion
    async login(email,password){}
```